code/modules/mod/mod_types.dm

```diff
@@ -303,3 +303,9 @@
 	theme = /datum/mod_theme/elite
 
 INITIALIZE_IMMEDIATE(/obj/item/mod/control/pre_equipped/syndicate_empty)
+
+/obj/item/mod/control/pre_equipped/honkerative
+	applied_skin = "honkerative"
+	theme = /datum/mod_theme/syndicate
+
+INITIALIZE_IMMEDIATE(/obj/item/mod/control/pre_equipped/honkerative)
```

Comment:
actually id just make this a subtype of syndicate empty since its a skin and itd inherit initialize immediate